steps/steps.py

```diff
@@ -0,0 +1,96 @@
+from behave import *
+from selenium import webdriver
+
+
+import environment
+import time
+
+
+class Calculator(object):
+    @step('that the Calculator app is loaded')
+    def app_is_loaded(context):
+        context.browser.get(
+            'http://seleniumsimplified.com/testpages/calculate.php'
+        )
+
+    @step('there is no current value in the calculator')
+    def check_current_value(context):
+        first_number_input = context.browser.find_element_by_xpath(
+            '//input[@id="number1"]'
+        )
+        second_number_input = context.browser.find_element_by_xpath(
+            '//input[@id="number2"]'
+        )
+
+        if first_number_input.get_attribute("value") != '':
```

Comment:
Jeszcze jedna ogólna uwaga - staraj się nie mieszać w jednym pliku/projekcie cudzysłowów pojedynczych i podwójnych. :) Wątpię, by ktoś Cię za to "oblał", ale to jest naruszenie tzw. dobrych praktyk. :)